src/main/java/frc/robot/Constants.java

```diff
@@ -0,0 +1,13 @@
+package frc.robot;
+
+public class Constants {
+
+	public static class NetworkTablesConstants {
+		public static final String TABLE_NAME = "datatable";
```

Comment:
delete this class and directly enter the string for the data table in RasberryPI.java